utils/cad_module_class.py

```diff
@@ -136,9 +155,21 @@ def vertex_indices_from_edges_tuple(self, bm, edge_tuple):
         > edge_tuple:   contains two edge indices.
         < returns the vertex indices of edge_tuple
         '''
-        k = lambda v, w: bm.edges[edge_tuple[v]].verts[w].index
-        return [k(i >> 1, i % 2) for i in range(4)]
+        e1, e2 = edge_tuple
+        bm_e1, bm_e2 = bm.edges[e1], bm.edges[e2]
+        return [bm_e1.verts[0].index, bm_e1.verts[1].index,
+                bm_e2.verts[0].index, bm_e2.verts[1].index]
```

Comment:
i don't know what possessed me to use bitwise operators and a lambda for this :)